docs/src/dsp.md

```diff
@@ -0,0 +1,12 @@
+# Signal Processing
+
+It is expected that these functions will be moved to a different package.
+
+```@docs
+FFTW.filt
+FFTW.filt!
+FFTW.deconv
+FFTW.conv
+FFTW.conv2
+FFTW.xcorr
```

Comment:
Yes, my plan is to move them to DSP, but for the sake of removing FFTW from Base I decided to just house them here temporarily, since they're the only Base functions which use FFTs. I figured it was easier to put them here first, since DSP will need a dependency on this package in order to provide those functions.